dashboard/app/models/game.rb

```diff
@@ -51,6 +51,10 @@ def self.custom_maze
   LEVEL_GROUP = 'level_group'.freeze
   PUBLIC_KEY_CRYPTOGRAPHY = 'public_key_cryptography'.freeze
 
+  def self.flappy
+    @@game_flappy ||= find_by_name("Flappy")
```

Comment:
I believe maze levels use the `CustomMaze` game, and artist levels use the `Custom` game.

```
Custom:turtle
CustomMaze:maze
```